opba-embedded-starter/src/main/java/de/adorsys/opba/starter/OpenBankingEmbeddedApplication.java

```diff
@@ -1,14 +1,20 @@
-package de.adorsys.obpa.starter;
+package de.adorsys.opba.starter;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-import de.adorsys.opba.tppbankingapi.config.EnableBankingApi;
+import de.adorsys.opba.tppbanking.api.config.EnableBankingApi;
 import de.adorsys.opba.consentapi.config.EnableConsentApi;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @SpringBootApplication
 @EnableBankingApi
 @EnableConsentApi
+@EntityScan(basePackages = "de.adorsys.opba.tppbanking.impl.domain.entity")
```

Comment:
maybe
EntityScan
EnableJpaRepositories
could be bound to EnableBankingPersistence configuration like on EnableConsentApi
This allows to select which application owns persistence and probably to reuse it

Reply:
ok